financial-connections/src/main/java/com/stripe/android/financialconnections/domain/IntegrityVerdictManager.kt

```diff
@@ -0,0 +1,20 @@
+package com.stripe.android.financialconnections.domain
+
+/**
+ * Manages the verdict of the integrity check. If the verdict is failed, the user will be switched to web flow.
+ *
+ * The scope of this is the application session. Subsequent launches of the AuthFlow within the hosting app after
```

Comment:
Should this just be an `object` then?

Reply:
Effectively it'd be the same, but its scope feels more "controlled" by relying on Dagger and the owning component to make it a singleton. Happy to change it if this does not make sense!